view/frontend/web/js/product/button.js

```diff
@@ -0,0 +1,52 @@
+define([
+    'jquery',
+    'underscore'
+],function ($, _) {
+
+    return function (config) {
+
+        $(config.buttonId).on('mouseover',function (e) {
+
+            var radioAndCheckbox = {};
+            var inputElements = [];
+
+            $.each($('.product-options-wrapper [aria-required="true"]'),function () {
```

Comment:
could you break out the inner jquery call `$('.product-options-wrapper [aria-required="true"]')` into a variable explaining what exactly it's grabbing? the class name gives a little insight, but it's not super clear